.bumpversion.cfg

```diff
@@ -0,0 +1,18 @@
+[bumpversion]
+current_version = 0.12.0dev
+commit = True
+
+parse = (?P<major>\d+)\.(?P<minor>\d+)\.(?P<patch>\d+)(?P<release>.*)?
+serialize = 
+	{major}.{minor}.{patch}{release}
```

Comment:
Instead of release should we use "prerelease"? Values I can think of could be rc-N, dev, alpha, beta, test.

Reply:
After reading the README more carefully, maybe it's best as it is to keep the API clear.